Source/CronosPlayUnreal/Public/DefiWalletCoreActor.h

```diff
@@ -306,113 +347,93 @@ class CRONOSPLAYUNREAL_API ADefiWalletCoreActor : public AActor {
     /**
      * Cosmos get balance.
      * @param success whether succeed or not
-     * @param output_message error message, "" if succeed
+     * @param Out callback
      */
     UFUNCTION(BlueprintCallable,
               meta = (DisplayName = "GetBalance", Keywords = "Wallet"),
               Category = "CronosPlayUnreal")
-    void GetBalance(FString address, FString denom, FString &output,
-                    bool &success, FString &output_message);
+    void GetBalance(FString address, FString denom,
+                    FGeneralCronosCallDelegate Out);
```

Comment:
Same as other comments, better off assigning it a dedicated delegate.

Reply:
ok